app/screens/gallery/footer/download_with_action/index.tsx

```diff
@@ -224,14 +232,17 @@ const DownloadWithAction = ({action, item, setAction}: Props) => {
             const path = getLocalFilePathFromFile(serverUrl, galleryItemToFileInfo(item));
             if (path) {
                 const exists = await fileExists(path);
-                let actionToExecute: (request: ClientResponse) => Promise<void>;
+                let actionToExecute: (response: ClientResponse) => Promise<void>;
                 switch (action) {
                     case 'sharing':
                         actionToExecute = shareFile;
                         break;
                     case 'opening':
                         actionToExecute = openFile;
                         break;
+                    case 'other':
```

Comment:
If this action executes the onDownload, why call it other and not downloading?

Reply:
I'll rename the function, the idea is to download the file as with all the actions but have this handled by a function passed as a prop instead of the component